### DISCUSSION TITLE
Custom route and legs pages

### DISCUSSION THREAD

**Discussion by Simleybob** [magknight/787aviatorsTracker#809] [Ideas]
These are the main pages that we all use when flying, they need a custom makeover especially that you are doing the less important ones already. People will complain big time if they are not done. Haven’t you made the graphics and other stuff for it already?

**Comment by aeroniemi** [comment]
doing RTE is probably actually impossible, as the default tech has the first line of the rte on page 1, which will make it rather challenging to replicate
legs could probably done, although it'll still be rather challening. 

if we do it at all, it'll probably be the last cdu stuff that we'll do.

**Reply by Simleybob** [reply]
For route: it’s not a matter of replicating the default stuff - replicate the real aircraft’s.

**Reply by aeroniemi** [reply]
it is a matter of replicating the default stuff - you won't be able to enter data, or update the page's content otherwise.

**Reply by Simleybob** [reply]
Ah right, so you can’t make a custom route page? How do other developers do it (if there are any?), how will ff do it?

**Reply by aeroniemi** [reply]
by writing a fully custom fmc?

**Reply by Simleybob** [reply]
Isn’t that what you are doing?

**Reply by aeroniemi** [reply]
no?

**Reply by fabian0hofmann** [reply]
![Bild_2022-04-18_200530810](https://user-images.githubusercontent.com/75206189/163852993-9cf16521-8d66-4620-bc9d-2e21dd3560b7.png)
also not true...
